packages/orbit-components/src/Portal/__tests__/index.test.js

```diff
@@ -1,43 +1,22 @@
 // @flow strict
 import * as React from "react";
-import { mount } from "enzyme";
+import { screen, render } from "@testing-library/react";
 
 import Portal from "../index";
 
-describe("Portal with renderInto", () => {
-  const target = document.createElement("div");
-  target.setAttribute("id", "portal");
+describe("Portal", () => {
+  it("should render into portal", () => {
+    const target = document.createElement("div");
+    target.setAttribute("id", "portal");
+    document.body?.appendChild(target);
 
-  // $FlowIssue
-  document.body.appendChild(target);
-  const component = mount(
-    <Portal renderInto="portal">
-      <p>Content</p>
-    </Portal>,
-  );
-  it("should match snapshot", () => {
-    expect(document.getElementById("portal")).toMatchSnapshot();
-    expect(component).toMatchSnapshot();
-  });
-  it("unmouted, should match snapshot", () => {
-    component.unmount();
-    expect(document.getElementById("portal")).toMatchSnapshot();
-    expect(component).toMatchSnapshot();
-  });
-});
-describe("Portal without renderInto", () => {
-  const component = mount(
-    <Portal>
-      <p>Content</p>
-    </Portal>,
-  );
-  it("should match snapshot", () => {
-    expect(document.getElementById("modal")).toMatchSnapshot();
-    expect(component).toMatchSnapshot();
-  });
-  it("unmouted, should match snapshot", () => {
-    component.unmount();
-    expect(document.getElementById("modal")).toMatchSnapshot();
-    expect(component).toMatchSnapshot();
+    render(
+      <Portal renderInto="portal">
+        <p>Content</p>
+      </Portal>,
+    );
+
+    expect(screen.getByText("Content")).toBeInTheDocument();
+    expect(document.getElementById("portal")).toBeInTheDocument();
   });
```

Comment:
You need to finish with `document.body?.removeChild(target)` at the end because Testing Library's `cleanup` removes only what was added by `render`.

Reply:
oh, yes, thanks :)